flowsa/flowby.py

```diff
@@ -673,7 +673,13 @@ def attribute_flows_to_sectors(
                     .rename(columns={'SourceName': 'MetaSources'})
                 )
             elif self.config['data_format'] in ['FBS']:
-                fb = grouped.sector_aggregation()  # convert to proper industry spec.
+                # ensure sector year of loaded FBS matches target sector year
+                fb = naics.convert_naics_year(
+                    grouped,
+                    f"NAICS_{self.config['target_naics_year']}_Code",
+                    grouped['SectorSourceName'][0])
+                # convert to proper industry spec.
+                fb = fb.sector_aggregation()
```

Comment:
@bl-young This change should check the FBS sector year and convert to target year if necessary - none of my FBS requires conversion so I will need to validate once you integrate "Detail_Use_year" FBS into the ghg/cap_hap methods

Reply:
ok just pushed them. i didn't test yet on top of your commit